distribution/adapters/fuse-adapter-zip/src/main/resources/licenses/keycloak/licenses.xml

```diff
@@ -125,11 +125,11 @@
     <dependency>
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpcore</artifactId>
-      <version>4.4.4</version>
+      <version>4.4.13</version>
```

Comment:
This is not correct, in Fuse we use https://github.com/keycloak/keycloak/blob/master/pom.xml#L1700-L1701

Reply:
Covered by https://github.com/keycloak/keycloak/pull/7463/commits/67dfcac8eb7be3ff5e4f74b78abeb040bfbe3965 change, PTAL